test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/EndToEndTestFixture.cs

```diff
@@ -51,10 +51,10 @@ protected EndToEndTestFixture(string rootPath, string testId, string extensionNa
             if (extensionName != null && extensionVersion != null)
             {
                 Host.SetNugetPackageSources("http://www.myget.org/F/azure-appservice/api/v2", "https://api.nuget.org/v3/index.json");
-                Host.InstallBindingExtension(extensionName, extensionVersion).Wait();
+                Host.InstallBindingExtension(extensionName, extensionVersion).Wait(TimeSpan.FromSeconds(30));
             }
 
-            Host.StartAsync().Wait();
+            Host.StartAsync().Wait(TimeSpan.FromSeconds(30));
```

Comment:
Adding these timeouts here to prevent future hangs if host initialization fails.